web/routing.go

```diff
@@ -141,6 +142,7 @@ func SetupRoutes(router *echo.Echo) error {
 	if config.IsDevRelease() {
 		router.GET("/dev/mails/:name", devMailsHandler)
 		router.GET("/dev/templates/:name", devTemplatesHandler)
+		sharings.Routes(router.Group("/sharings"))
```

Comment:
Since we don't have feature flags yet and I don't want the sharing stuff to go in production, I've put the routing for it only for dev release. It is obviously only temporary.